### PR TITLE
#162055931 Users should be able to update article 

### PR DESCRIPTION
**Pull request by koiic**
#### What does this PR do?
     This PR implement functions to update an article
#### Description of Task to be completed?
      1. Write implementation for a user to update their article
#### Any background context you want to provide?
        N/A
### What are the relevant pivotal tracker stories?
       #162055931
#### Questions:
        N/A

### CODE REVIEW

File: src/components/article/EditArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+      response
+        ? swal('Success', 'Article published Successfully', 'success')
+        : swal('Failed', errorMessages, 'error')}
+      );
+  };
+
+  render() {
+    if(this.state.editMode === true){
+      if(this.props.publishedArticle.processing){
+        console.log(this.props.publishedArticle.Articles)
```
Comment (KaiserPhemi) [inline src/components/article/EditArticle.jsx]: #### Critical
Remove all logging.

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
@koiic 
Implement feedback and rebase.

File: src/components/article/EditArticle.jsx (hunk 1 above)
Comment (koiic) [reply src/components/article/EditArticle.jsx]: done

File: src/action/article/updateArticle.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// third party libraries
+import http from 'axios';
```
Comment (e0works) [inline src/action/article/updateArticle.js:2]: Please add a blank line here

File: src/action/tag/tag.js [hunk 3]
```diff
@@ -78,13 +85,13 @@ export const fetchSingleTag = (tagName) => (dispatch) => {
   };
 
   return http.get(`http://localhost:5000/api/v1/search/tag/${tagName}`, options).then((response) => {
-    console.log('RESPONSE', response)
-    dispatch(fetchTagSuccess(response.data))
+    console.log('RESPONSE', response);
```
Comment (e0works) [inline src/action/tag/tag.js:88]: Please remove this logging statement

File: src/reducers/article/updateArticle.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+// modules
```
Comment (e0works) [inline src/reducers/article/updateArticle.js:1]: please, change this comment here to `state` or `store`

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**